tests/unit/example.spec.js

```diff
@@ -2,11 +2,12 @@ import { shallowMount } from '@vue/test-utils'
 import HelloWorld from '@/components/HelloWorld.vue'
 
 describe('HelloWorld.vue', () => {
-  it('renders props.msg when passed', () => {
-    const msg = 'new message'
-    const wrapper = shallowMount(HelloWorld, {
-      propsData: { msg }
-    })
-    expect(wrapper.text()).toMatch(msg)
+  const wrapper = shallowMount(HelloWorld, {
+    mocks: {
+      $t: msg => msg
```

Comment:
We are not using translations for now, so this shouldn't be necessary, please try to remove it.